common/changes/@bentley/itwin-viewer-react/toolAdmin_2021-04-01-18-14.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@bentley/itwin-viewer-react",
+      "comment": "Added support for ToolAdmin",
+      "type": "patch"
```

Comment:
since this is a new feature, it should be a minor version bump

Reply:
I thought so too, but CONTRIBUTING.md said "always choose "patch" as the change type." I'll run `rush change` again

Reply:
you can just edit the file directly (change "patch" to "minor"). We'll have to update CONTRIBUTING. That was likely a copy/paste error. Thanks for finding

Reply:
Ahh I already ran it again. I'll delete and update